### DISCUSSION TITLE
ANCOM-BC Vignette Question

### DISCUSSION THREAD

**Discussion by carlthebas** [FrederickHuangLin/ANCOMBC#97] [Q&A]
Thank you for all your hard work in creating and maintaining ANCOM-BC. Even as a novice R user, I've managed to get it installed and had an OK time applying the code from the vignette to my own data. As I'm looking at differential abundant taxa between healthy and diseased samples, I've been trying to use the vignette to create a visualization similar to that of BMI for the primary result. 

My question is in the visualization for BMI, is obese the reference group for the comparison? If so, am I correct in assuming taxa in red were more abundant in obese, while taxa in teal were less abundant in lean? 

If this is an incorrect analysis of that figure, what code should I look at for using ANCOM-BC to visualize what taxa are more or less abundant in my samples when comparing healthy vs diseased? Thanks again!  

Carl

**Comment by FrederickHuangLin** [comment]
Hi @carlthebas,

Please find below my responses to your questions:

> My question is in the visualization for BMI, is obese the reference group for the comparison?

Yes. Note that by default, levels of a categorical variable in R are sorted alphabetically. For instance, if you have three groups of BMI: `lean`, `overweight`, and `obese`, the reference level will be `lean` by default. To manually change the reference level, for instance, setting `obese` as the reference level, use: `data$bmi = factor(data$bmi, levels = c("obese", "overweight", "lean"))` You can verify the change by checking:
`levels(data$bmi)`

> If so, am I correct in assuming taxa in red were more abundant in obese, while taxa in teal were less abundant in lean?

Yes. Since the reference level is `lean`, the effect size will be `obese` - (minus) `lean`

I am aware that many people are confused about the reference level in ANCOMBC outputs, so in the latest version, I have updated the vignette for a more detailed description of managing the reference levels. 

I just pushed the changes to the Bioconductor branches. It might take a few business days for the package to become available (but the `devel` version is ready here!).

Best,
Huang
